container/docker/factory.go

```diff
@@ -172,9 +172,12 @@ func (self *dockerFactory) DebugInfo() map[string][]string {
 	return map[string][]string{}
 }
 
+var (
+	version_regexp_string = "(\\d+)\\.(\\d+)\\.(\\d+)"
+	version_re            = regexp.MustCompile(version_regexp_string)
```

Comment:
Putting it in the function means the RE gets wastefully compiled every time the function is called. In this case probably not called more than once, but normally best to move it outside so trying to keep it consistent.